infra-l2-waf/template.yaml

```diff
@@ -0,0 +1,255 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: CloudFormation template defines Web ACL resources
+
+Parameters:
+  Environment:
+    Description: "The environment we're deploying into."
+    Type: "String"
+    Default: dev
+    AllowedValues:
+      - "dev"
+      - "build"
+      - "staging"
+      - "integration"
+      - "production"
+
+Conditions:
+  IsProdLikeEnvironment: !Or
+    - !Equals [!Ref Environment, staging]
+    - !Equals [!Ref Environment, integration]
+    - !Equals [!Ref Environment, production]
+
+Mappings:
+  PlatformConfiguration:
+    dev:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    build:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    staging:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    integration:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    production:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+
+# If a rule exists in ExcludedRules, then it's action is set to Count, instead of the default action for that group.
+# To upgrade a rule to follow the group action, please comment it out, so that it becomes active.
+# @TODO We need to find a better way to set this up.
+
+Resources:
+  webAcl:
+    Type: 'AWS::WAFv2::WebACL'
+    Properties:
+      Description: Web ACL for API Gateway
+      Name: !Sub '${AWS::StackName}-${Environment}-acl'
+      DefaultAction:
+        Allow: {}
+
+      Rules:
```

Comment:
just to confirm - are the first 4 rule groups being launched in Block mode from the start